.github/workflows/ci.yml

```diff
@@ -323,6 +323,12 @@ jobs:
         run: go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.49.0
       - run: make lint
       - run: make test
+      - name: Check library version
+        if: |
+          !github.event.pull_request.head.repo.fork &&
+          startsWith(github.ref, 'refs/tags/go/mcap/v')
+        working-directory: go/cli/check-tag
+        run: make build && bin/check-tag
```

Comment:
Should we put this in the `go-release-cli` step instead? I see that go-release-cli has `needs: [go]`, however it still does its own build and as far as I can see doesn't actually depend on the output of the `go` job. So it seems a little nicer for this validation to be in the job that needs it (go-release-cli).

Reply:
I now see that this is not relevant to the cli because it's for `go/mcap/vX` rather than `go/mcap-cli/vX`.